src/components/bookAddForm.js

```diff
@@ -0,0 +1,16 @@
+const BookAddForm = () => (
+  <div className="bookFormDiv">
+    <h3>ADD NEW BOOK</h3>
+    <form className="formFields">
+      <input className="bookTtlInp" type="text" id="inputBookName" placeholder="Book title" />
+      <select className="bookCateg" type="dropdown" id="inputBookName" placeholder="Category">
+        <option value="action">Fiction</option>
+        <option value="non-fiction">Non-Fiction</option>
+        <option value="fantacy">Fantasy</option>
+      </select>
```

Comment:
- [ ] As required for the project, kindly ensure that your form to create new books has `title` and `author` inputs. As it mentioned in the requirements, you should replace this code `<select className="bookCateg" type="dropdown" id="inputBookName" placeholder="Category"><option value="action">Fiction</option> <option value="non-fiction">Non-Fiction</option>....</select>` with an `<input />` tag for the `Author`.
![Screenshot from 2023-06-15 22-42-31](https://github.com/Nahid1911/bookstore_using_react/assets/83042742/0bf63b05-f324-4e99-a66d-fbb1f9f5c51e)

Reply:
@tresorsawasawa  Thank you for your review, the has been resolved.